gammapy/spectrum/tests/test_spectrum_extraction.py

```diff
@@ -14,21 +14,24 @@
 from ...datasets import gammapy_extra
 from ...image import ExclusionMask
 from ...spectrum import SpectrumExtraction, SpectrumObservation
-
+import logging
+import numpy as np
 
 @pytest.mark.parametrize("pars,results", [
-    (dict(containment_correction=False), dict(n_on=95,
-                                              sigma=19.70,
-                                              aeff=549861.8268659255 * u.m ** 2,
-                                              ethresh=0.4353353353 * u.TeV)),
-    (dict(containment_correction=True), dict(n_on=95,
-                                             sigma=19.70,
-                                             aeff=393356.18322397786 * u.m ** 2,
-                                             ethresh=0.6179 * u.TeV)),
+    (dict(containment_correction=False), dict(n_on=172,
```

Comment:
I think it does,
if you're interested: the reco_binning was fixed to [1, 10] TeV before,  that doesn't really make sense IMO. If you're interested in this energy range you should restrict the fit range (as is now possible).
So the extra counts probably come from [ -inf, 1) AND (10, + inf]